src/DGmethods/test/Euler/isentropic_vortex_standalone.jl

```diff
@@ -217,6 +223,9 @@ let
   ll == "ERROR" ? Logging.Error : Logging.Info
   logger_stream = MPI.Comm_rank(mpicomm) == 0 ? stderr : devnull
   global_logger(ConsoleLogger(logger_stream, loglevel))
+  @static if Base.find_package("CUDAnative") !== nothing
+    device!(MPI.Comm_rank(mpicomm) % length(devices()))
```

Comment:
We should probably allow this to come from the scheduler.

Reply:
This certainly isn't a long term solution.

In my experience the scheduler should set the devices you can see on a node so you are at least protected there. Not sure the scheduler sets the node each MPI rank can see, and what about machines that you run on that don't have schedulers?

This also has the problem that it assumes that all nodes have the same numbers of GPUs, that the number of ranks on a node is the same as the number of GPUs, that mpiranks are consecutive across GPUS, etc..

In the past we've used an all gather on host name then assigned GPUs that way with the assumption that number of GPUs on a node is the same as the number of ranks.

Better solutions are certainly welcome, but this should work in the near-term I think(?).

@lcw probably has thoughts too.